internal/lib/checkpoint.go

```diff
@@ -54,8 +81,67 @@ func (c *ContainerServer) ContainerCheckpoint(ctx context.Context, opts *Contain
 		return "", fmt.Errorf("failed to checkpoint container %s: %w", ctr.ID(), err)
 	}
 	if opts.TargetFile != "" {
-		if err := c.exportCheckpoint(ctr, specgen.Config, opts.TargetFile); err != nil {
-			return "", fmt.Errorf("failed to write file system changes of container %s: %w", ctr.ID(), err)
+		logrus.Debugf("Trying to save the checkpoint as %s\n", opts.TargetFile)
+		if strings.HasPrefix(opts.TargetFile, "/") || !strings.Contains(opts.TargetFile, "/") {
```

Comment:
Is this check for verifying that the path is absolute?

Reply:
I added a long comment what this does. It is basically a workaround for the limited choices the CRI API gives us.